tor/strings/en_US.yml

```diff
@@ -112,6 +113,30 @@ responses:
       If the footer is correct, have you tried replying "Done" again after one-two minutes? Sometimes, freshly posted transcriptions are not seen by the bot, so trying again should fix the issue.
 
       If it still doesn't work, it could be the sign of another bug; let us know by [sending a Modmail](https://www.reddit.com/message/compose?to=%2Fr%2FTranscribersOfReddit&subject=Transcription%20Issue&message=) so we can take a look!
+    
+    promotion_text:
+      grafeas-jade: |
+        Superb performance! You just transcribed more than 10000 posts.
+      grafeas-topaz: |
+        Outstanding! You are just halfway from hitting jade. Keep up the good work.
+      grafeas-ruby: |
+        Excellent! You have just hit the 4 digit number 1000. Lets Go!!!
+      grafeas-diamond: |
+        You are as tough as a diamond. What more to say about your good work.
+      grafeas-golden: |
+        Yipee!! Your work is so golden.
+      grafeas-purple: |
+        Congratulations!! You have hit the 250 mark. Keep going!
+      grafeas-teal: |
+        Century!! You have just hit the magic number 100.
+      grafeas-green: |
+        Congratulations!! You have achieved your first rank.
+      grafeas-initiate: |
+        Hooray!! You just made your first transcription.
```

Comment:
I think it would be good to have the number of transcriptions included in every message here, in case the volunteer is not familiar with the ranks.

Some of the numbers seem to be in the wrong message, e.g. Ruby is 2500 transcriptions and not 1000 (that's diamond). Here is a full list of the ranks and their transcription counts:

- 1 - 49: Initiates
- 50 - 99: Green
- 100 - 249: Teal
- 250 - 499: Purple
- 500 - 999: Gold 
- 1,000 - 2,499: Diamond
- 2,500 - 4,999: Ruby
- 5,000 - 9,999: Topaz
- 10,000+: Jade